redirects-list.json

```diff
@@ -59,5 +59,5 @@
 
   "^/doc/studio/view-settings$                                                            /doc/studio/user-guide/views/view-settings",
 
-  "^/(.+)/$                                                                               /$1"
+  "^/([^/].+)/$                                                                           /$1"
```

Comment:
I would suggest we make it a bit more robust. Catch it in the places where we for those redirects. E.g. in the 

`/src/server/middleware/redirects/index.js`

do something like:

```
redirectLocation = redirectLocation.replace(/([^:]\/)\/+/g, "$1")
```

wdyt?

we need to prevent this from happening at the core level, so if someone makes a mistake in the redirects list we won't go back to the regression.

Ideally we should also add a test for this (we already have tests for redirects)

Reply:
I think that's a fine idea, especially the test, since if the regression happens it won't be obvious otherwise.

Reply:
Yep, I would check if that place alone is enough though. I think we consume redirects link in a few places (?).